test/metabase/automagic_dashboards/core_test.clj

```diff
@@ -208,6 +208,110 @@
             (perms/grant-collection-readwrite-permissions! (perms-group/all-users) collection-id)
             (test-automagic-analysis (db/select-one Card :id card-id) 8)))))))
 
```

Comment:
Step 1: Add some reasonable tests about the actual contents of cards being generated for specific cases. In this PR, the focus is on a model created from a table.